MeshLib/Node.h

```diff
@@ -79,13 +70,16 @@ class Node : public MathLib::Point3dWithID
 		return Node(_x[0]-v[0], _x[1]-v[1], _x[2]-v[2]);
 	}
 
-protected:
-	/**
-	 * Add an element the node is part of.
-	 * This method is called by Mesh::addElement(Element*), see friend definition.
-	 */
+	/// Update coordinates of a node.
+	/// This method automatically also updates the areas/volumes of all connected elements.
+	virtual void updateCoordinates(double x, double y, double z);
```

Comment:
Before methods `updateCoordinates()` and `addElement()` where protected. Now they are in the public interface. Was this change intended?